README.md

```diff
@@ -1 +1,44 @@
-# Project1
\ No newline at end of file
+# Project1
+Hello Microverse
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. The goal here is to master all of the tools and best practices I learned . All about gitflow , linters.
+
+set up a new repository and prepare it for development using best practices.
```

Comment:
- [ ] I would kindly suggest you to customize the description text so that it described your project. Currently it starts from `In this project, you will set up a "Hello world" repository.`  which sound like describing the task. :thinking: